style.css

```diff
@@ -0,0 +1,120 @@
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+  width: 375px;
+  height: 823px;
+}
+
+body {
+  background-color: white;
+  margin: 0;
+  padding: 0;
+  line-height: 2rem;
+  color: #152081;
+  font-family: poppins, sans-serif;
+  font-weight: 400px;
+  font-size: 15px;
+}
+
+h1,
+h2 {
+  margin: 0;
+  font-weight: 700;
+  color: black;
+}
+
+.header-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
+.logo-info {
+  font-family: poppins, sans-serif;
+  font-style: bold;
+  font-size: 18pt;
+  font-weight: 700;
+  line-height: 20pt;
+  line-height: 74%;
+  text-align: left;
+  left: 12px;
+  top: 10px;
+  vertical-align: center;
+  fill: solid #000001;
+  text-decoration: none;
+  color: #6070ff;
+  margin: 0;
+  padding: 0;
+}
+
+.logo-vector-btn {
+  border: 0;
+  background-color: inherit;
+}
+
+.welcome {
+  text-align: center;
+}
+
+.container-1 {
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
```

Comment:
- [ ] [OPTIONAL] Great job 👍 . Kindly make sure the spacing between the navbar and the text is similar to the one in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205395963-187ad9c0-d79f-43cf-a5c0-3968b5e6420c.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205396382-bb3522ad-1837-4c6e-9238-ed2ca9a191c1.png)

- [ ] [OPTIONAL] Kindly make sure your text looks as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205396246-65ce4c4b-1d2f-4ffb-a499-9fd3370c4b6a.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205396279-4c346c5b-6486-48b0-900f-bc55a91176a0.png)

Reply:
@Ugboaja-Uchechi I will surely do that. Thank you